src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyAloneOrSingleline.java

```diff
@@ -138,14 +138,14 @@ void doDoubleBraceInitialization() {
             add("AB21/X");
             add("YYLEX");
             add("AR5E");
-        }});  //it's ok, can't be formatted better
+        }});  //violation
 
         foo23(new java.util.HashSet<String>() {{
             add("XZ13s");
             add("AB21/X");
             add("YYLEX");
             add("AR5E");
-        }});} //violation
+        }});} //2 violations
```

Comment:
why two violations ?
I do not see it in UT expected results. Please explain.

Reply:
> I do not see it in UT expected results

It is there.
````
+            "148:9: " + getCheckMessage(MSG_KEY_LINE_ALONE, "}", 9),
+            "148:13: " + getCheckMessage(MSG_KEY_LINE_ALONE, "}", 13),
````